src/components/ExtraDataRightMenu/ExtraDataMenu.js

```diff
@@ -25,6 +25,8 @@ function ExtraDataMenu(props) {
 			<h3 className="extraData-title font-bold primary-color">Show data for:</h3>
       <Dropdown 
         arrowClassName='arrow'
+        // arrowClosed='arrow'
```

Comment:
Do we still need the commented out bit here?

Reply:
No, I will remove that. Thanks for catching it.